x-pack/plugins/enterprise_search/public/applications/analytics/index.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import { isVersionMismatch } from '../../../common/is_version_mismatch';
+import { InitialAppData } from '../../../common/types';
+import { VersionMismatchPage } from '../shared/version_mismatch';
+
+import { AnalyticsOverview } from './components/analytics_overview/analytics_overview';
+
+import { ROOT_PATH } from './routes';
+
+export const Analytics: React.FC<InitialAppData> = (props) => {
+  const { enterpriseSearchVersion, kibanaVersion } = props;
+  const incompatibleVersions = isVersionMismatch(enterpriseSearchVersion, kibanaVersion);
+
+  const showView = () => {
```

Comment:
nitpick: using const vars as React blocks inside of a react module is an anti-pattern we've been moving away from, as it messes with React's diff'ing logic. Better to just do this inside your codeblock, or create a new component outside of this component for this.

Reply:
Agreed! Good catch. For now I will just inline but a nice tech debt ticket would be to spin this out as its a common pattern (showing incompatible versions) across all the other ent-search plugins